app/Console/Commands/WaitingLists/ProcessPendingRemovals.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands\WaitingLists;
+
+use App\Console\Commands\Command;
+use App\Events\Training\AccountRemovedFromWaitingListDueToActivity;
+use App\Events\Training\AccountWithinFiveDaysOfWaitingListRemoval;
+use App\Models\Training\WaitingList;
+use Carbon\Carbon;
+
+class ProcessPendingRemovals extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'waitinglists:processpendingremovals';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Processing waiting list accounts marked for automatic removal';
+
+    /**
+     * Executes all necessary console commands.
+     */
+    public function handle()
+    {
+        WaitingList::all()->filter(function ($waitingList) {
+            return $waitingList->isAtcList();
```

Comment:
I'd suggest since you're using this filter for ATC lists in more than one place,  you create a local scope: https://laravel.com/docs/9.x/eloquent#local-scopes

Reply:
Hi @AxonC 

I have created a local scope for this exact purpose in https://github.com/VATSIM-UK/core/pull/2804 as their was a similar requirement in that PR, and a local scope was suggested by @atoff.

Makes sense to utilize that here, however this change is blocked until 2804 is merged.